README.md

```diff
@@ -38,6 +38,49 @@ EVE Docker container `lfedge/eve:<version>` is used to produce these deployment
 
 Try running `docker run lfedge/eve` to get the most up-to-date help message.
 
+The versions in the tag of `lfedge/eve:<version>` contain information as to which hypervisor and target architecture they
+support. The options are:
+
+* architecture: `amd64`, `arm64`, `riscv64`
+* hypervisor: `kvm`, `xen`, `mini`
+
+Note that not all hypervisors are supported on all architectures.
+
+For example:
+
+* `docker run lfedge/eve:8.11.0-kvm-arm64`: installer for 8.11.0 using kvm on arm64
+* `docker run lfedge/eve:8.11.0-xen-arm64`: installer for 8.11.0 using xen on arm64
+* `docker run lfedge/eve:8.11.0-xen-amd64`: installer for 8.11.0 using xen on amd64
+* `docker run lfedge/eve:8.11.0-mini-riscv64`: installer for 8.11.0 using mini on riscv64
```

Comment:
None of these get you the installer; they just print a usage string. Did you intend to add "installer" to the line?

Reply:
Sort of. There are different installer commands (`installer_raw`, `installer_net`, etc.), so I didn't want to pick. I can add something that indicates a command should be there.